modules/nf-core/qualimap/bamqc/tests/main.nf.test

```diff
@@ -0,0 +1,36 @@
+nextflow_process {
+
+    name "Test Process QUALIMAP_BAMQC"
+    script "../main.nf"
+    process "QUALIMAP_BAMQC"
+    tag "modules"
+    tag "modules_nfcore"
+    tag "qualimap"
+    tag "qualimap/bamqc"
+
+    test("homo_sapiens [BAM]") {
+
+        when {
+            params {
+                outdir   = "$outputDir"
+            }
+            process {
+                """
+                input[0] = [
+                    [ id:'test', single_end:false ], // meta map
+                    file(params.test_data['homo_sapiens']['illumina']['test_paired_end_sorted_bam'], checkIfExists: true)
+                ]
+                input[1] = []
+                """
+            }
+        }
+
+        then {
+            assertAll (
+                { assert process.success },
+                { assert path(process.out.results.get(0).get(1)).exists() },
```

Comment:
in the pytest tests there a lot more outputs were snapped. can you please reproduce the pytests.

Reply:
I copy pasted the logic from the rnaseq tests, but I'll have a look, thanks

Reply:
I probably have myself to blame for that, so I won't even `git blame` to discover that it was me all along

Reply:
ok, so actually I am indeed to blame for the qualimap/rnaseq tests, but they were minimalist

Reply:
so, so I'm not going to check all the files from the pytest, that was overboard, I think we only need to test the existence of the html file and the snapshot of the txt file